src/renderServer.ts

```diff
@@ -14,21 +14,18 @@ import {validateRenderData} from './validate';
  */
 export function renderServer(config: ConfigService) {
   const app = express();
+  const renderRoutes = express.Router();
 
   Sentry.init({
     dsn: process.env.SENTRY_DSN,
-    integrations: [
-      new Integrations.Express({app}),
-      new Sentry.Integrations.Http({tracing: true}),
```

Comment:
Do we lose anything from this

Reply:
we won't see the request to get the chartcuterie config in the breadcrumbs

Reply:
cool